File: check_table.py
import sys, os, re
sys.stdout.reconfigure(encoding='utf-8')

ws = r'C:\Users\fireh\WorkBuddy\20260326144402\tmg-website'

# These 12 articles need table and banner
# Let me check the reference article to see how tables are used
fp = os.path.join(ws, 'blog', 'baidu-advertising-complete-guide-2026.html')
with open(fp, 'r', encoding='utf-8') as f:
    content = f.read()

start = content.find('<article class="article-content reveal">')
end = content.find('</article>', start)
article = content[start:end]

# Find table usage
tables = list(re.finditer(r'<table class="article-table">', article))
print(f'Found {len(tables)} article-tables in reference')

# Show first table
if tables:
    t = tables[0]
    table_end = article.find('</table>', t.start())
    table_html = article[t.start():table_end+8]
    print('First table:')
    print(table_html[:500])

File: check_table.js
const fs = require("fs");
const c = fs.readFileSync(String.raw`C:\Users\fireh\WorkBuddy\20260326144402\tmg-website\ja\blog\cpm-is-rising-bad.html`, "utf8");
const idx = c.indexOf("[table]");
if (idx >= 0) {
  console.log("[table] FOUND at position " + idx);
  console.log("Context: " + c.substring(Math.max(0, idx - 100), idx + 100));
} else {
  console.log("[table] NOT found in deployed file");
}
// Also check if article-table exists
console.log("\narticle-table present: " + /class="article-table">/.test(c));
